firebase/FirebaseConfig.ts

```diff
@@ -31,8 +31,7 @@ export const storage = getStorage(firebaseApp);
 // be running via `yarn fb:emu` first before starting the app via `yarn dev`)
 const isEmulatorMode =
   process.env.NEXT_PUBLIC_IS_EMULATOR_MODE?.toLowerCase() === 'true';
-const isEmulatorEnvironment =
-  process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test';
```

Comment:
Using emulators for dev only (removed `test` here), running in test is pointless and generates a warning for the auth service when running the unit tests, which don't require the emulators anyhow